src/HtsSystemContractJson.sol

```diff
@@ -81,6 +82,35 @@ contract HtsSystemContractJson is HtsSystemContract {
         _initTokenInfo(json);
     }
 
+    function _initTokenRelationships(address account) internal override {
+        if (relationshipsInitialized[account]) {
+            // Already initialized
+            return;
+        }
+
+        bytes32 slot = _isAssociatedSlot(account);
+        try mirrorNode().fetchTokenRelationshipOfAccount(vm.toString(account), address(this)) returns (string memory json) {
+            string memory notFoundError = "{\"_status\":{\"messages\":[{\"message\":\"Not found\"}]}}";
```

Comment:
We can use `keyExistsJson` instead of comparing with the error as we did it in other places.